### PR TITLE
[LA-2025] Initial commit for Los Angeles 2025

### PR DESCRIPTION
**Pull request by omargohan**
**Summary of changes:**

- Initial commit for Los Angeles 2025

**Tested:**

- Locally using Hugo

### CODE REVIEW

**Comment by brynak** [comment]
Happy Labor Day Weekend, everyone! We're excited to get DevOpsDays Los Angeles 2025 launched. Please let us know how we can help. :)

File: data/events/2025/los-angeles/main.yml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  - name: contact
+  - name: conduct
+  - name: about
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
```
Comment (phrawzty) [inline data/events/2025/los-angeles/main.yml:62]: There are six entries on your team_members list, but seven addresses in your mail distribution list. These two lists must line up. Please either add the "missing" person to this list, or send an email to info@devopsdays.org with the name _and email address_ of the person to _remove_ from your distribution list.

**Review by phrawzty — CHANGES_REQUESTED** [review]

File: data/events/2025/los-angeles/main.yml (hunk 1 above)
Comment (brynak) [reply data/events/2025/los-angeles/main.yml:62]: thanks, @phrawzty . email sent to info@

Comment (omargohan) [reply data/events/2025/los-angeles/main.yml:62]: @phrawzty I have made some changes to the list. Thank you!

File: content/events/2025-los-angeles/propose.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
++++
+<br>
+<hr>
+<center><h3>Important Dates for DevOpsDaysLA 2025</h3></center>
+<br>
+{{< cfp_dates >}}
+<center><strong>DevOpsDaysLA 2025 Conference: 7th March 2025</strong></center>
+<hr>
+<center>
+  <h2><a href="https://www.socallinuxexpo.org/scale/22x/cfp" style="font-size: 1.5em; padding: 5 10px;text-decoration: none; border-radius: 5px;">>>> Submit your proposal here <<<</a></h2>
```
Comment (phrawzty) [inline content/events/2025-los-angeles/propose.md:14]: The `<` and `>` symbols have a specific meaning in HTML, thus [page compilation will fail](https://github.com/devopsdays/devopsdays-web/actions/runs/10760428131/job/29838371448?pr=14518#step:7:13) if they're not treated properly. There are few ways to handle this situation; this is one suggestion.

```suggestion
  <h2><a href="https://www.socallinuxexpo.org/scale/22x/cfp" style="font-size: 1.5em; padding: 5 10px;text-decoration: none; border-radius: 5px;">&gt;&gt;&gt; Submit your proposal here &lt;&lt;&lt;</a></h2>
```

**Review by phrawzty — CHANGES_REQUESTED** [review]

File: data/events/2025/los-angeles/main.yml (hunk 1 above)
Comment (phrawzty) [reply data/events/2025/los-angeles/main.yml:62]: I confirm that the lists line up and have responded in email as well.

**Comment by adrianmoisey** [comment]
Heads up that you may want to update the redirect here: https://github.com/devopsdays/devopsdays-web/blob/0085e7afb888527503d099766e579742b5be4d76/static/_redirects#L69

File: content/events/2025-los-angeles/propose.md (hunk 2 above)
Comment (omargohan) [reply content/events/2025-los-angeles/propose.md]: Fixed, thank you for the suggestion.

**Review by phrawzty — APPROVED** [review]

**Comment by brynak** [comment]
Thanks, Everybody!

**Pull request merged**